In-house patch to remove unnecessary dependencies from cloudbase-init's
requirements file. The specific reasons are as follows:

Babel			Not really used by cloudbase-init

iso8601			Not really used by cloudbase-init

untangle		Not applicable (for specific services we do not support)

pyserial		Not applicable (Python Serial Port Extension - we do not support this)

comtypes		Not applicable to Solaris (and lack of PEP 426 support)

pymi			Not applicable to Solaris (and lack of PEP 426 support)

pywin32			Not applicable to Solaris (and lack of PEP 426 support)

tzlocal			Not applicable to Solaris (and lack of PEP 426 support)

wmi			Not applicable to Solaris (and lack of PEP 426 support)

--- cloudbase-init-1.1.6/requirements.txt
+++ cloudbase-init-1.1.6/requirements.txt
@@ -1,19 +1,10 @@
 pbr>=2.0.0
-iso8601
 netaddr>=0.7.6
-pyserial
 oslo.config
 oslo.log
-Babel>=1.3
 oauthlib
 netifaces
 PyYAML
 requests
-untangle==1.2.1
 jinja2
 cryptography
-pywin32;sys_platform=="win32"
-comtypes;sys_platform=="win32"
-pymi;sys_platform=="win32"
-wmi;sys_platform=="win32"
-tzlocal;sys_platform=="win32"
